core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1155,6 +1202,13 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
       .setServiceAccountPrivateKeyFromPemFile(new java.io.File(pemFile))
       .build()
 
+  def getBucketServiceAccountCredentials: GoogleCredentials =
```

Comment:
The `Credential` that is used elsewhere is deprecated; in order to use the current metrics API I need to use `Credentials` instead, which needs the service account key in the form of a JSON.